### ISSUE TITLE
Uncaught Error: ENOENT, open 'c:\Users\cstead\Documents\Events.com\rui\app\spec\services\composite\commitOrder-spec.js'

### ISSUE THREAD

**Issue by cmstead** [atom/atom#5420]
[Enter steps to reproduce below:]

1. Open a file that differs from one branch to another
2. Switch branches -- keeping the file open
3. An error opens.  I'm not clear on the reason. It's intermittent.

**Atom Version**: 0.176.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: ENOENT, open 'c:\Users\cstead\Documents\Events.com\rui\app\spec\services\composite\commitOrder-spec.js'

```
At events.js:85

Error: ENOENT, open 'c:\Users\cstead\Documents\Events.com\rui\app\spec\services\composite\commitOrder-spec.js'
  at Error (native)

```

### Commands

```
```

### Config

```json
{
  "core": {},
  "editor": {
    "fontFamily": "Source Code Pro",
    "fontSize": 17,
    "lineHeight": 1.4,
    "preferredLineLength": 128,
    "scrollPastEnd": true,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v1.1.0
editorconfig, v0.3.0
html-helper, v0.2.3
open-last-project, v0.1.1
pretty-json, v0.3.1
remember-session, v0.5.1
slash-closer, v0.7.1

# Dev
No dev packages
```

**Comment by benogle** [comment]
Does this happen every time you use your repro steps? Or is it intermittent? Please quit atom completely and run `atom --safe`. Is the issue still there?

Thanks for the steps to reproduce. Almost no one actually enters them.

**Comment by cmstead** [comment]
It's intermittent. I'll give 'atom --safe' a try tomorrow and see what I
get.  It's hard to say, though as the error just kind of went away as the
day went on.  It's really a strange one. Some days I see it all the time,
then I might go several days without an issue at all. Sorry I couldn't help
more. : (

Also, I'm glad to enter steps to reproduce a problem. I'm a developer, so I
understand needing steps to reproduce an issue. Otherwise a bug report is
basically useless. ; )

Chris Stead
cmstead@gmail.com
http://www.chrisstead.com

**Comment by 50Wliu** [comment]
Closing in favor of #6210.
